CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* ResolveNames and ResoveNamesW operations perform a more complete Ambiguous Name Resolution search.
```

Comment:
I really like you add user defined changes, but I'm afraid this is not so user oriented. Could you explain what this improves the user experience in address list?